tcms/wsgi.py

```diff
@@ -36,15 +36,15 @@
 # This application object is used by any WSGI server configured to use this
 # file. This includes Django's development server, if the WSGI_APPLICATION
 # setting points here.
-_application = get_wsgi_application()
+_APPLICATION = get_wsgi_application()
```

Comment:
It looks like you can move the get_wsgi_application call down in the application() function